packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -38,11 +39,18 @@ export type ChatScreenProps = {
   onRenderMessage?: (messageProps: MessageProps, defaultOnRender?: MessageRenderer) => JSX.Element;
   onRenderTypingIndicator?: (typingUsers: CommunicationParticipant[]) => JSX.Element;
   onFetchParticipantMenuItems?: ParticipantMenuItemsCallback;
+  styles?: MessageThreadStyles;
```

Comment:
I'm confused. This prop isn't being set by the `ChatComposite` in this PR. Nor is this exposed in some form in the `ChatComposite` API. How will this get set?

Reply:
should be piped through now, thanks for checking this. I think my test code was just injecting styles into the ChatComposite component which was incorrect.

Reply:
I was thinking that was intentional to not be exposed on the composite API - are we supporting arbitrary styling ? (based on previous breaking changes/customizations conversations I would assume we set good defaults but not let user apply arbitrary styling inside the composite)